src/nostr_manager/mod.rs

```diff
@@ -473,23 +470,15 @@ impl NostrManager {
         &self,
         signer: impl NostrSigner + 'static,
         pubkey: PublicKey,
+        contact_list_pubkeys: Vec<PublicKey>,
```

Comment:
Currently I know we're doing this for "contacts and self" however I think that's a mistake and we have to do it for all users